### PR TITLE
Feat/Footer

### PR DESCRIPTION
**Pull request by joavelar**
Draft Mode :)
Mobile:
Screenshot 2024-04-03 at 3 20 31 PM
Desktop:
Screenshot 2024-04-03 at 3 22 21 PM

### CODE REVIEW

File: client/src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { FaInstagram, FaLinkedin, FaTiktok } from "react-icons/fa";
+import styles from "./Footer.module.css";
+
+export default function Footer() {
```
Comment (elimanzo) [inline client/src/components/Footer/Footer.tsx:4]: There's some inconsistent tabbing, I believe we're sticking with 2 tab space instead of 4

**Review by elimanzo — CHANGES_REQUESTED** [review]
Just a small feedback. Also, for PRs some screenshots of the mobile + web would be great too.

File: client/src/components/Footer/Footer.tsx (hunk 1 above)
Comment (joavelar) [reply client/src/components/Footer/Footer.tsx:4]: Fixed

**Comment by jipatino98** [comment]
Good work Jose! I'll will merge your branch

**Pull request merged**